### ISSUE TITLE
MCQ Selected answer button is only identified visually by color (WCAG 1.4.1) [QAC]

### ISSUE THREAD

**Issue by ftourangeau** [MeD-DMC/Adapt-accessibilityfixes#220]
**COURSE:**
https://esdc-edsc-adapt.canadacentral.cloudapp.azure.com/preview/5dc446fd27f08836496b6b9e/63a32079b75e454dafed1e1d/
*******************************************************************************
**Page:**
Lesson 1: Overview of electronic Passport Next Generation
Lesson 2: Domestic Print
Lesson 3: Observation Print
Lesson 4: Manual QA
Lesson 5: Reports

**Description:**
The selected answer (disabled radio button) 
   • does not have any other visual indication from surrounding not selected answer buttons (disabled radio buttons) other than color AND
   • the contrast difference between the selected and non-selected button colors is not at least 3:1

Color must not be the only way to convey information on the page. The color may not be seen by users with color deficiencies. In this case, providing the information conveyed with color through another visual means ensures users who cannot see color can still perceive the information.

To improve the accessibility of the buttons, provide other visual means, for example:
   • provide at least 3:1 color contrast between the selected button and other buttons AND
   • add a text to the button name, e.g.: "All of the above (selected answer)" or something similar
![image](https://user-images.githubusercontent.com/93537804/231586042-e31f7978-efb5-400c-a875-097fbc3f386c.png)
*******************************************************************************
